andela/king-kong-ah-backend
# #166414806 Re-arrange folder structure

#### What does this PR do?

- These PR Re-arranges the folder structure 

#### Description of Task to be completed?

- Move test folder to root server folder
- Modify package.json test script to run from the new path

#### How should this be manually tested?

  - run `git clone https://github.com/EmmanuelSage/EpicMail-React.git`
  - Check the folder structure
  - The test folder should reside in the server folder

#### What are the relevant pivotal tracker stories?

- PT Story link - [#166414806](https://www.pivotaltracker.com/story/show/166414806)
